packages/amplify-e2e-tests/custom-resources/custom-cdk-stack-compile-error.txt

```diff
@@ -0,0 +1,21 @@
+import * as cdk from '@aws-cdk/core';
+
+/**
+ * Base amplify stack class
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export class cdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    /* Do not remove - Amplify CLI automatically injects the current deployment environment in this input parameter */
+    // eslint-disable-next-line no-new
+    new cdk.CfnParameter(this, 'env', {
+      type: 'String',
+      description: 'Current Amplify CLI env name',
+    });
+
+    // intentional compilation error below:
+    compile error
+  }
+}
```

Comment:
these, and other txt files, are being used to overwrite the 'overrides.ts' files, so that we can simulate a customer putting incorrect code (code that doesn't compile, or throws at runtime)

Reply:
have we chosen txt files to not have IDE complain about problems inside ?

Reply:
yes, they are just basic copy/paste so extension doesn't matter - only the compiler will complain so i used txt